offerzen/global/employers/employers@7.js

```diff
@@ -0,0 +1,1618 @@
+        label: label || 'Company Sign Up / Employer Landing Page',
+        category: category || 'Core',
+        source: source || 'Demand Sign Up',
+      });
+    }
+
+    function onSubmitOriginalCompanyLeadForm(token, e) {
+      originalLeadFormButton.attr('disabled', true);
+      let initialButtonValue = originalLeadFormButton.attr('value');
+      let dataWait = originalLeadFormButton.attr('data-wait');
+      originalLeadFormButton.attr('value', dataWait);
+      // get the value of the report_source query parameter should it be present and forward it onto form lead submission for analytics
+      let searchParams = new URLSearchParams(window.location.search);
+      const formData = new FormData(originalLeadForm[0]);
+      const formProperties = Object.fromEntries(formData.entries());
+      const roleTypes = getRoleTypes(formProperties);
+
+      if (originalLeadForm.parsley().validate()) {
+        originalLeadForm.find('.js-missing-fields').hide();
+        trackSubmission();
+        $.ajax({
+          type: 'POST',
+          url: '/company/form_leads',
+          data: JSON.stringify(
+            Object.assign({}, formProperties, {
+              workplace_policy: roleTypes.join(','),
+              referrer: location.href,
+              report_source: searchParams.get('report_source'),
+              'g-recaptcha-response-data': {
+                webflow: token,
+              },
+            })
+          ),
+          contentType: 'application/json',
+          headers: {
+            Accept: 'application/json',
+          },
+          success: function (data) {
+            if (data.user_id) {
+              window.location.href = data.redirect_url;
+            }
+          },
+          // Reset form
+          error: function (data) {
+            originalLeadFormButton.attr('disabled', false);
+            originalLeadFormButton.attr('value', initialButtonValue);
+          },
+        });
+      } else {
+        $('.js-missing-fields').show();
+        originalLeadFormButton.attr('disabled', false);
+        originalLeadFormButton.attr('value', initialButtonValue);
+      }
+    }
+
+    function updateSubscribeToHiringInsightsField() {
+      subscribeToCompanyNewsletter = originalLeadForm.find(
+        '#subscribe_to_company_newsletter'
+      );
+      subscribeToCompanyNewsletter.on('change', function () {
+        originalLeadForm.find('#subscribe_to_hiring_insights').val(this.value);
+      });
+    }
+
+    // ------------------------------------------------------------
+    // Phone Number Input
+    // ------------------------------------------------------------
+
+    function normaliseCharacters(term) {
+      return term
+        .toLocaleLowerCase()
+        .normalize('NFD') // Creates separate bytes for characters and diacritical marks
+        .replace(/[\u0300-\u036f]/g, ''); // Replaces any diacritical marks in the string
+    }
+
+    // Country Data Set
+    const phoneNumberCountries = [
```

Comment:
Please move this to the top of phone number input for clarity (just after the comment)